infra/stacks/dns.ts

```diff
@@ -0,0 +1,27 @@
+const ZONE_NAME = "hebo.ai";
+const DEV = "dev";
+
+const createZoneIfNotExists = async (zoneName: string) => {
+  try {
```

Comment:
Instead of try / catch you can use "import" arg to take over an existing zone:

Example: https://github.com/sst/console/blob/dev/infra/dns.ts